src/test/java/Homework21.java

```diff
@@ -0,0 +1,39 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework21 extends BaseTest {
+    @Test
+    public void renamePlaylist() throws InterruptedException {
+        String updatedPlaylistMessage = "Updated playlist \"Rename.\"";
+
+        logIn();
+        Thread.sleep(2000);
+        doubleClickPlaylist();
+        Thread.sleep(2000);
+        enterNewPlaylistName();
+        Thread.sleep(2000);
+        //Assert
+        Assert.assertEquals(renameSuccessMessage(), updatedPlaylistMessage);
```

Comment:
Please, delete `throws InterruptedException`

Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.